### DISCUSSION TITLE
WiFI SSW101B driver with mainline

### DISCUSSION THREAD

**Discussion by ddB0515** [linux-chenxing/linux-chenxing.org#52] [General]
Hello all :wave: 

As I understand this is barely used/mentioned around but a lot of modules comes with this WiFi chip SSW10X from SigmaStar and as few drivers is on github as source but only for stock kernel released with SDK (4.9.84)

Have anyone tried to compile make it work with mainline Linux kernel?? 
Or this is locked/intended to work with stock kernel?

**Comment by fifteenhex** [comment]
I quickly tried to compile it but it didn't just work. I think it'll need a bit of work to convert over to mainline.
The chip is actually a rebrand of a chip from another company so I think the best plan would be to try to find a newer version of the original chip driver.

**Reply by ddB0515** [reply]
Can you please tell me from what chip is rebranded so I could take a look (I assume Mediatek...)

**Reply by fifteenhex** [reply]
The notes I have are here:

https://github.com/linux-chenxing/linux-chenxing.org/blob/master/index.md#non-soc-chips
